pkg/types/aws/machinepool.go

```diff
@@ -107,6 +110,12 @@ type EC2RootVolume struct {
 	// +optional
 	IOPS int `json:"iops"`
 
+	// Throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	Throughput int64 `json:"throughput"`
```

Comment:
```suggestion
	// Throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
	// 
	// This is only applicable to gp3 volume type.
	//
	// When omitted, this means no opinion, and the platform is left to
	// choose a reasonable default, which is subject to change over time.
	//
	// +kubebuilder:validation:Minimum:=125
	// +kubebuilder:validation:Maximum:=2000
	// +optional
	Throughput int64 `json:"throughput"`
```

WDYT? We should probably define a kube marker for valid min and max range to match what we validate in `pkg/types/aws/validation/machinepool.go`.